src/Illuminate/Bus/Queueable.php

```diff
@@ -95,7 +140,14 @@ public function dispatchNextJobInChain()
     {
         if (! empty($this->chained)) {
             dispatch(tap(unserialize(array_shift($this->chained)), function ($next) {
+                if (! in_array('Illuminate\Bus\Queueable', class_uses_recursive($next))) {
+                    throw new \Exception('Trying to dispatch an object that is not Queueable');
+                }
```

Comment:
I don't understand why this is here. It should be removed. If the methods don't exist, we will get an error anyways.